### PR TITLE
Update create monika_clothing topic

### PR DESCRIPTION
**Pull request by Blu3pulsar**
Created a topic about Monikas school uniform and future clothing preferences

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4847,6 +4847,38 @@ label monika_coffee:
...
+    m 2r "Also school uniforms are not as comfortable as people think." 
+    m 4n "At least not for us girls." 
+    m 1q "In winter my legs are freezing."
+    m 1q "And in summer I'm sweating very fast in this blazer."
+    m 5j "On the other hand, people like them and think they look cute. Ahaha~"
+    m 1a "But what about you [player], would you like me to change my outfit sometime?"  
+    menu:
+        "Yes":
+            m 2k "Ahaha, I knew this was a good idea." 
+	        m 2j "Maybe I will find some new clothes in my code soon."
```
Comment (monikLover) [inline Monika After Story/game/script-topics.rpy]: Fix indents.

Comment (Blu3pulsar) [reply Monika After Story/game/script-topics.rpy]: quick question: To fix stuff, just edit the file again as before ?

Comment (Blu3pulsar) [reply Monika After Story/game/script-topics.rpy]: sry never used this kollab system before

**Comment by synnnical** [comment]
While there is nothing wrong with what the topic is talking about, a majority of the lines within the topic feel way too robotic and stiff. The only way I'd only be able to help fix that is by changing the entire topic around and I'm not sure you're okay with that, given you already added expressions, and adding or taking away new lines would probably mess things up for you.

I'd encourage you to go over it and polish it a bit more in terms of word usage, grammar placement, and the flow (how the entire thing rolls of Monika's tongue). For the time being, I will close it until you decide to resubmit it.

**Pull request closed without merging**